examples/fscloud/main.tf

```diff
@@ -0,0 +1,33 @@
+##############################################################################
+# Resource group
+##############################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.4"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+##############################################################################
+# Create CIS instance and add domain
+##############################################################################
+module "cis_instance" {
+  source            = "../../modules/fscloud"
+  service_name      = "${var.prefix}-example"
+  resource_group_id = module.resource_group.resource_group_id
+  tags              = []
+  plan              = "standard-next"
+  domain_name       = var.domain_name
+  add_dns_records   = true
+  dns_record_set = [
+    {
+      type    = "A"
+      name    = "test-example"
+      content = "1.2.3.4"
+      proxied = true
+    }
+  ]
+  add_glb = false
```

Comment:
I was able to create a CIS instance when both `add_dns_records` & `add_glb` were `false`. So shouldn't there be a validation to make sure either `DNS records` or `GLB` were inputted by the user since for activating DDOS we need DNS or GLB proxied records?

Reply:
Not exactly. There are use-cases where an user do not need to add any DNS/GLB. So if no record, then DDoS will not play any role. 
If records are there, then it must be proxied else does not matter.